scripts/build-boshrelease-pipelines.sh

```diff
@@ -48,3 +48,5 @@ setup_release_pipeline rds-broker alphagov/paas-rds-broker-boshrelease master
 setup_release_pipeline datadog-for-cloudfoundry alphagov/paas-datadog-for-cloudfoundry-boshrelease master
 setup_release_pipeline logsearch-for-cloudfoundry alphagov/paas-logsearch-for-cloudfoundry gds_master
 setup_release_pipeline paas-haproxy alphagov/paas-haproxy-release master
+setup_release_pipeline graphite CloudCredo/graphite-statsd-boshrelease master
```

Comment:
Not sure about automatically building from upstream. What if the build fails? Do we want to instead only want to use releases they make?